.circleci/test_examples.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+DIR="/tmp/workspace/json"
+cd $1 || exit
+EXAMPLES="example"
+if [ "$(ls -A $EXAMPLES)" ]; then
+    echo "Found examples"
+else
+    echo "Found no examples in $EXAMPLES"
+    exit 1
+fi
+
+FILES=$(/bin/ls -1 $EXAMPLES|grep -E '.json')
+cd $EXAMPLES || exit 2
+for f in $FILES
+do
+	ajv test -s "$DIR/$1/schema.json" -d $f --valid
```

Comment:
Is it documented how to run this script? Feels like you need a ajv lib.

Reply:
Yes, you do need it, but it is only (or at least primarily) meant for CircleCI. If it could be solved in some more generic way that did not rely on a NPM lib I would be very happy to consider that. I could probably document it in some way (maybe inside the script itself).

Reply:
Updated the script, should give you a hint now if ajv is not installed on how to install.